src/components/winning-conditions.js

```diff
@@ -0,0 +1,13 @@
+//covers winning sets of tiles
+const lines = [
```

Comment:
I'd name this more explicit (and change the file name to match).

I'd also move this to a folder src/utils since it's not a React component

```suggestion
const winningConditions = [
```

Reply:
initially, the file name was using kebab case, named 'winning-conditions', do you want me to change the file name using camel case?